core/storage/base_model/gae_models.py

```diff
@@ -47,8 +47,8 @@
 )
 
 EXPORT_POLICY = utils.create_enum(  # pylint: disable=invalid-name
-    'CONTAINS_USER_DATA',
-    'NOT_APPLICABLE',
+    'EXPORTED',
+    'NOT_EXPORTED'
```

Comment:
Sorry about this, but would it be OK to change NOT_EXPORTED to NOT_APPLICABLE? On looking at the PR, I'm finding that it would be easier to tell the difference (and therefore catch errors/violations) if the enums looked less similar to each other.

Reply:
Changed.